src/components/events/events-card/events-card.jsx

```diff
@@ -120,12 +120,12 @@ const EventsCard = ({ event }) => {
       onClick={() => toggleAccordion()}
     >
       <Grid container spacing={3}>
-        <Grid item xs={12} sm={4} md={3} lg={3}>
-          <EventCardDate />
-        </Grid>
         <Grid item xs={12} sm={8} md={7} lg={7}>
           <EventCardHeader />
         </Grid>
+        <Grid item xs={12} sm={4} md={3} lg={3}>
+          <div> </div>
```

Comment:
If we're not planning on rendering anything here maybe we can get rid of it? (and adjust the grid rules)